### PR TITLE
[Security] Bump crypto libraries

### PR DESCRIPTION
**Pull request by sarahalsmiller**
### Description
Update crypto libraries 

- HOLDING OFF ON MERGING THIS

### Testing & Reproduction steps
- ci passes 

### PR Checklist

* [ ] updated test coverage
* [ ] external facing docs updated
* [ ] appropriate backport labels added
* [ ] not a security concern

### CODE REVIEW

**Review by dduzgun-security — DISMISSED** [review]
LGTM 👍

**Review by xwa153 — APPROVED** [review]

File: .release/security-scan.hcl [hunk 1]
```diff
@@ -84,6 +84,7 @@ binary {
 	triage {
 		suppress {
 			vulnerabilities = [
+				"GO-2022-0635", // github.com/aws/aws-sdk-go@v1.55.5
```
Comment (dduzgun-security) [inline .release/security-scan.hcl:87]: Were there any reasons why we suppress this alert?

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @sarahalsmiller! a backport is missing for this PR [22001] for versions [1.15,1.18,1.19,1.20] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @sarahalsmiller! a backport is missing for this PR [22001] for versions [1.15,1.18,1.19,1.20] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Comment by hc-github-team-consul-core** [comment]
:mega: Hi @sarahalsmiller! a backport is missing for this PR [22001] for versions [1.15,1.18,1.19] please perform the backport manually and add the following snippet to your backport PR description:
```
<details>
	<summary> Overview of commits </summary>
		- <<backport commit 1>>
		- <<backport commit 2>>
		...
</details>
```

**Pull request merged**